src/config/feature-flags.ts

```diff
@@ -13,7 +13,6 @@ const launchdarklyClient = LaunchDarkly.init(envVars.LAUNCHDARKLY_KEY || "", {
 
 export enum BooleanFlags {
 	MAINTENANCE_MODE = "maintenance-mode",
-	SIMPLER_PROCESSOR = "simpler-processor",
```

Comment:
Please try to avoid changes that don't address the PR topic. I know that it's annoying to create a separate PR for these changes, but it gives so much more confidence when reviewing the PR.